pages/admin/dashboard/toolBuilder.tsx

```diff
@@ -21,11 +23,21 @@ import { SelfCheckQuestionCard, ToolHomePage } from "@components";
 import { useRouter } from "next/router";
 import axios from "axios";
 
-export const getServerSideProps = async () => {
-    return {
-        props: {}, // will magically be passed to the page component as props
-    };
+export const getServerSideProps: GetServerSideProps = async ({ req, res }) => {
+    if (process.env.NODE_ENV == "production") {
+        const authProps = await isAuthenticated(req, res, "/redirect", true);
```

Comment:
Can we add a // TODO: change redirect to login page (once we have a login page that's deployed)